pom.xml

```diff
@@ -636,7 +646,7 @@
       <properties>
         <gpg.keyname>FAB0BE712374BFC3D26DA5B43A7041C227272DD8</gpg.keyname>
         <gpg.executable>gpg2</gpg.executable>
-        <gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>
+        <!--<gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>-->
```

Comment:
I think this needs to not be commented out

Reply:
Oh I see.. this causes an error in my environment... any link to a documentation on this?

Reply:
Oh well. did uncommented this. Feels like black magic somehow as this definitely shouldn't result in an error in any environment unless proper guidelines are provided explaining the Why's and How's.